### DISCUSSION TITLE
Instagram Support

### DISCUSSION THREAD

**Discussion by cewong2** [TumblThreeApp/TumblThree#273] [Ideas]
Twitter was added last year, just wondering what is the possibility of getting Instagram support as well? That would make this a trifecta of an app.

**Comment by thomas694** [comment]
At the moment we don't have any plans to add Instagram support. But we will put it on the wish list.

Maybe someone knows a good program for windows.

**Comment by fusionmc-evilscoop** [comment]
look at 4k stogram @cewong2
